pkg/sync/v6/data/PrometheusRule.monitoring.coreos.com/openshift-monitoring/prometheus-k8s-arorules.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: prometheus-k8s-arorules
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: aro-sync.rules
+    rules:
+    - alert: SyncPodDown
+      annotations:
+        description: Sync Pod is down
+        summary: Sync Pod is down
+      expr: |
+        absent(up{service="sync"} == 1)
+      for: 10m
+      labels:
+        severity: warning
```

Comment:
It might be more understandable if the following were used as severity values, `page` and `ticket`. However I understand this might not be straightforward due to the out-of-the-box rules use `warning` and `critical`. Whichever way is chosen, I would expect `ticket` or `warning` to create a SEV3/SEV4 ticket while `critical` and `page` end up as SEV1/SEV2 which triggers a page.

Reply:
I should document that, as this is the idea. Warnings should not wake people up (Sev3/4) while criticals should (Sev2).